app-backend/api/Dockerfile

```
@@ -1,8 +1,7 @@
-FROM openjdk:8-jre-alpine
+FROM openjdk:11-slim
 
 RUN \
-      addgroup -S rf \
-      && adduser -D -S -h /var/lib/rf -s /sbin/nologin -G rf rf
+    adduser --system --disabled-password --home /var/lib/rf --shell /sbin/nologin --disabled-password --group rf
```

This deviates a bit from what is currently being done with Histo and the prediction API. I think that it would be good to align, or otherwise provide justification for the deviations.

---

I made the `adduser` command align exactly with what was being done in the Alpine Linux variant.

I'm also fine with having it align with `backsplash-server/Dockerfile`, which is also based on Debian:

```suggestion
    adduser --home /var/lib/rf --shell /sbin/nologin --disabled-password --gecos "" rf
```